src/vma/sock/sock-redirect.cpp

```diff
@@ -1792,10 +1643,6 @@ void vma_epoll_create(int epfd, int size)
 extern "C"
 int epoll_create(int __size)
 {
-	BULLSEYE_EXCLUDE_BLOCK_START
-	if (!orig_os_api.epoll_create)	get_orig_funcs();
-	BULLSEYE_EXCLUDE_BLOCK_END
-
 	do_global_ctors();
```

Comment:
@rafiw What about _do_global_ctors()_ ? can we call it during VMA startup and remove the calls from sock-redirect.cpp?